health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/SearchChecklist.js

```
@@ -37,45 +37,60 @@ const SearchChecklist = () => {
-      select: (data) => {
-        return data?.MdmsRes;
-      },
+        schemaCode: "HCM.CHECKLIST_TYPES"
+      }
     },
+    changeQueryName: "HCM"
   };
-  const { isLoading, data: HCM, isFetching } = Digit.Hooks.useCustomAPIHook(reqCriteriaResource);
+  const { isLoading, data: HCM, isFetching } = Digit.Hooks.useCustomAPIHook(reqCriteria1);
+  useEffect(() => {
+    let data = HCM?.mdms;
```

Biome

[error] This let declares a variable that is only assigned once.

'data' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)